src/components/form-helpers/form-helpers.js

```diff
@@ -100,20 +100,24 @@ export const FormWithCaptcha = ({
     // If challenge window is closed, we do not have feedback, so, by the moment,
     // we will keep the submit button disabled.
     // See more details in https://stackoverflow.com/questions/43488605/detect-when-challenge-window-is-closed-for-google-recaptcha
-    formikProps.setSubmitting(false);
-    const result = await verifyCaptcha();
-    formikProps.setSubmitting(true);
-    if (result.success) {
-      await originalOnSubmit(
-        { ...values, captchaResponseToken: result.captchaResponseToken },
-        formikProps,
-      );
+    if (process.env.NODE_ENV === 'test') {
```

Comment:
Nice approach to fix this issue for testing.

Reply:
You could try this way that you are doing: use the captcha form as it were a normal form for testing, or test that the captcha is fired if you use a captcha form. Perhaps a middle point approach could be, to have a flag to disable the captcha by code (instead of validating with test env) to allow both scenarios. Perhaps something like this:
```suggestion
    if (process.env.NODE_ENV === 'test' && process.env.REACT_APP_DISABLE_CAPTCHA) {
```

Reply:
I was reviewing my changes and it seems to be that this changes are not necessaries. 😄

Reply:
I have deleted the last comment. If you read it, ignore it.

Reply:
No leí todo, pero no está bueno leer variables de entorno en los js, tampoco verificar si el entorno es de test.

Entiendo que puede ser difícil, que tal vez no sea el momento de hacerlo y que entonces sea razonable dejar este if, pero es una ventana rota, por lo menos deberíamos dejar un comentario con un TODO.

Lo ideal sería por injección de dependencias inyectar un dummy del captcha.

Reply:
Thanks @andresmoschini  for the suggestion, are you up to this change @lucianaDilizio ?

Reply:
>  cbernat 20 minutes ago Member
> Thanks @andresmoschini for the suggestion, are you up to this change @lucianaDilizio ?

No 😅. I will try to do it here, but maybe it would be better to do this in another PR.

Reply:
If you are willing to do this in another PR perhaps all tests should be there as well.

Reply:
I think that it is fine doing it here, the tests are addinmg safety. 

What do you think about simply adding a TODO comment by the moment?